lms/static/scripts/frontend_apps/components/test/BasicLtiLaunchApp-test.js

```diff
@@ -204,6 +204,28 @@ describe('BasicLtiLaunchApp', () => {
       );
     });
 
+    it('does not create a second auth window when clicked twice', async () => {
```

Comment:
It isn't immediately obvious what "when clicked twice" refers to. Perhaps `when "Authorize" button is clicked twice`?